internal/batches/executor/changeset_specs_test.go

```diff
@@ -8,22 +8,15 @@ import (
 	"github.com/sourcegraph/batch-change-utils/overridable"
 	"github.com/sourcegraph/src-cli/internal/batches"
 	"github.com/sourcegraph/src-cli/internal/batches/git"
-	"github.com/sourcegraph/src-cli/internal/batches/graphql"
 )
 
 func TestCreateChangesetSpecs(t *testing.T) {
-	srcCLI := &graphql.Repository{
-		ID:            "src-cli",
-		Name:          "github.com/sourcegraph/src-cli",
-		DefaultBranch: &graphql.Branch{Name: "main", Target: graphql.Target{OID: "d34db33f"}},
-	}
-
 	defaultChangesetSpec := &batches.ChangesetSpec{
-		BaseRepository: srcCLI.ID,
+		BaseRepository: testRepo1.ID,
```

Comment:
I extracted the repository into a global var for the tests because every test duplicated the same 5 lines and I'd rather have a single place where test fixtures are defined.

The pattern itself is relatively common, I think. This is the crudest version of it. On the fanciest end of the range you'd have test helpers that generate parameterized test data, etc. But since these here are read-only I think it's fine to just share them globally in the tests.

Reply:
That makes sense! Thanks for explaining!